addons/mail/models/mail_alias.py

```diff
@@ -150,11 +150,34 @@ def _clean_and_check_unique(self, name):
 
         catchall_alias = self.env['ir.config_parameter'].sudo().get_param('mail.catchall.alias')
         bounce_alias = self.env['ir.config_parameter'].sudo().get_param('mail.bounce.alias')
+        alias_domain = self._default_alias_domain()
```

Comment:
Actually you could directly call the ir config parameter so that it is inlined with other parameters